tests/recipes/cad1/task1/test_evaluate.py

```diff
@@ -101,14 +100,14 @@ def test_make_song_listener_list():
                 }
             },
             {
-                "left_drums": 0.149880148,
-                "right_drums": 0.143182857,
-                "left_bass": 0.140449345,
-                "right_bass": 0.181374074,
-                "left_other": 0.132401105,
-                "right_other": 0.164211137,
-                "left_vocals": 0.121260627,
-                "right_vocals": 0.126605279,
+                "left_drums": 0.140304064,
```

Comment:
Some of these scores have changed significantly. Was that expected?

Reply:
Yes, because we add the RMS factor to the score.
Do, the score now is a combination of HAAQI and RMS of the silence parts.

Thai change of the score was part of the discussion with Trevor